### PR TITLE
add upgrade note about default consul protocol

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: source/upgrade/upgrade_notes.rst [hunk 1]
```diff
@@ -4,6 +4,15 @@
 Upgrade notes
 *************
 
+20.01
+=====
+
+* The default protocol configured for consul is now ``HTTP`` instead of ``HTTPS`` since it's only
+  available on ``localhost``. The ``HTTPS`` remains available via the port ``8501``.
```
Comment (fredericlepied) [inline source/upgrade/upgrade_notes.rst]: we need to change the installer to implement this behaviour. Right now it's either 8500/http or 8501/https. Not both.

Comment (fblackburn1) [reply source/upgrade/upgrade_notes.rst]: On upgrade the config file from ansible will be erased by the postinst of wazo-consul-config
https://github.com/wazo-platform/wazo-consul-config/blob/master/debian/wazo-consul-config.postinst#L54
Maybe we should discuss of the wanted behavior

**Comment by fblackburn1** [comment]
@fredericlepied Can we discuss about it (or it's already done and I don't remember)? We need to fix it for the 20.01 release

File: source/upgrade/upgrade_notes.rst (hunk 1 above)
Comment (sduthil) [reply source/upgrade/upgrade_notes.rst]: I implemented this default behavior in Ansible: https://github.com/wazo-platform/wazo-ansible/pull/60

**Pull request merged**